pkg/util/paramtable/component_param.go

```diff
@@ -2452,7 +2452,7 @@ func (p *dataNodeConfig) init(base *BaseTable) {
 	p.FlowGraphMaxQueueLength = ParamItem{
 		Key:          "dataNode.dataSync.flowGraph.maxQueueLength",
 		Version:      "2.0.0",
-		DefaultValue: "1024",
+		DefaultValue: "16",
```

Comment:
it's 1024 in milvus.yaml, the same as pipeline of querynode, better to unify all DefaultValue to 16??
![image](https://github.com/milvus-io/milvus/assets/51370125/bc226516-d055-4353-b089-426dba8d6170)
![image](https://github.com/milvus-io/milvus/assets/51370125/1a52ed4f-82df-4866-a62d-0ef0b67802e4)